gokart/info.py

```diff
@@ -25,16 +25,27 @@ def make_tree_info(task, indent='', last=True, details=False):
         indent += '|  '
     name = task.__class__.__name__
     result += f'({is_complete}) {name}[{task.make_unique_id()}]'
+
+    if compress:
```

Comment:
@vaaaaanquish 
[IMO]
Since just suppress printing rather than creating compression tree, how about use `abbr`, `abbreviation` or `redundant` for argument name

Reply:
Good point.

It was abbr at first.
https://github.com/m3dev/gokart/pull/192/commits/8abd114e1d25be95bfc35b0f6c4244c09a8f4245
I think abbr is good too.

Reply:
Wow sorry for confusing.